modules/react/combobox/lib/ComboboxInput.tsx

```diff
@@ -77,7 +79,10 @@ export const useComboboxInput = composeHooks(
           model.events.setWidth(event.currentTarget.clientWidth);
         }
       },
-      value: model.state.value,
+      value:
+        model.state.selectedIds.length > 0 && model.state.cursorId !== ''
+          ? model.state.items[currentValue].textValue
+          : model.state.value,
```

Comment:
I had to do this because it was rendering the `id` as the value instead of the `textValue` in the input.